crates/sui-framework/packages/bridge/sources/committee.move

```diff
@@ -182,6 +199,51 @@ module bridge::committee {
         abort 0
     }
 
+    #[test]
+    #[expected_failure(abort_code = ESignatureBelowThreshold)]
+    fun test_verify_signatures_with_blocked_committee_member() {
+        let committee = setup_test();
+        let msg = message::deserialize_message(hex::decode(TEST_MSG));
```

Comment:
rename TEST_MSG to `BLOCKLIST_COMMITTEE_MEMBER_MSG` and add comment to explain what it does?

Reply:
TEST_MSG is a token transfer message, this is a test message used for signature verification